programmers/난이도별/level01.키패드_누르기/SSSOy.java

```diff
@@ -0,0 +1,43 @@
+class Solution {
+    
+    static int keypad[][] = {{0, 0}, {0, 1}, {0, 2}, {1, 0}, {1, 1}, {1, 2}, {2, 0}, {2, 1}, {2, 2}, {3, 0}, {3, 1}, {3, 2}};
+    static int fingers[][] = {{3, 0}, {3, 2}};
+    
+    public String solution(int[] numbers, String hand) {
+        String answer = "";
+        
+        for(int i = 0; i < numbers.length; i++) {
+            switch(numbers[i]) {
+                case 1 : case 4 : case 7:
+                    answer += "L";
+                    fingers[0] = keypad[numbers[i] - 1];
+                    break;
+                case 3 : case 6 : case 9:
+                    answer += "R";
+                    fingers[1] = keypad[numbers[i] - 1];
+                    break;
+                default :
+                    answer += middle(numbers[i], hand);
+            }
+        }        
+        return answer;
+    }
```

Comment:
```suggestion
        for(int number; numbers) {
            switch(number) {
                case 1 : case 4 : case 7:
                    answer += "L";
                    fingers[0] = keypad[number - 1];
                    break;
                case 3 : case 6 : case 9:
                    answer += "R";
                    fingers[1] = keypad[number - 1];
                    break;
                default :
                    answer += middle(number, hand);
            }
        }        
        return answer;
    }
```
이렇게 forEach 형태로 해주면 여러가지로 신경쓸게 많이 없어집니다!